tests/e2e/workloads/couchbase/test_couchbase_pod_respin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, workloads, ignore_leftovers
+from tests.disruption_helpers import Disruptions
+from ocs_ci.utility import utils
+
+log = logging.getLogger(__name__)
+
+
+@workloads
+@ignore_leftovers
+class TestCouchBaseWorkload(E2ETest):
+    """
+    Deploy an CouchBase workload using operator
+    """
+    @pytest.fixture()
+    def cb_setup(self, couchbase_factory_fixture):
+        """
+        Creates couchbase workload
+        """
+        self.cb = couchbase_factory_fixture(
+            replicas=3, run_in_bg=True, skip_analyze=True)
+
```

Comment:
Need to change (pep8)
        self.cb = couchbase_factory_fixture(
            replicas=3, run_in_bg=True, skip_analyze=True
)